srml/support/procedural/src/lib.rs

```diff
@@ -40,23 +40,47 @@ use proc_macro::TokenStream;
 /// }
 /// ```
 ///
-/// Declaration is set with this header `(pub) trait Store for Module<T: Trait> as Example`
-/// with `Store` a (pub) trait generated associating each storage to the `Module` and
-/// `as Example` setting the prefix used for storages of this module. `Example` must be unique:
-/// another module with same name and same inner storage item name will conflict.
+/// Declaration is set with the header `(pub) trait Store for Module<T: Trait> as Example`,
+/// with `Store` a (pub) trait generated associating each storage item to the `Module` and
+/// `as Example` setting the prefix used for storage items of this module. `Example` must be unique:
+/// another module with the same name and the same inner storage item name will conflict.
 ///
 /// Basic storage consists of a name and a type; supported types are:
 ///
 /// * Value: `Foo: type`: Implements [StorageValue](../srml_support/storage/trait.StorageValue.html).
-/// * Map: `Foo: map type => type`: implements [StorageMap](../srml_support/storage/trait.StorageMap.html)
-/// * Linked map: `Foo: linked_map type => type`: Implements [StorageMap](../srml_support/storage/trait.StorageMap.html)
-/// and [EnumarableStorageMap](../srml_support/storage/trait.EnumerableStorageMap.html).
-/// * Double map: `Foo: double_map u32, $hash(u32) => u32;`: Implements `StorageDoubleMap` with `$hash` representing a
-/// choice of hashing algorithms available in [`Hashable` trait](../srml_support/trait.Hashable.html).
+/// * Map: `Foo: map hasher($hash) type => type`: Implements [StorageMap](../srml_support/storage/trait.StorageMap.html)
+///   with `$hash` representing a choice of hashing algorithms available in the
+///   [`Hashable` trait](../srml_support/trait.Hashable.html).
 ///
-///   /!\ Be careful when choosing the hash function, malicious actors could craft second keys to lower the trie.
+///   `hasher($hash)` is optional and its default is blake2_256.
```

Comment:
```suggestion
///   `hasher($hash)` is optional, and its default is `blake2_256`.
```

Reply:
Yeah, should be `blake2_256`, but don't agree with the comma